From 1309c91b9692c9e0fb693fd1783a5488f062a51c Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Thu, 19 Jul 2012 15:21:53 +0800
Subject: [PATCH 332/442] ENGR00215123 - SPDC : fix segmetation fault issue.

- Correct the case number to avoid program generate
  segmetation fault after all cases completion.

Signed-off-by: Fugang Duan  <B38611@freescale.com>
---
 test/mxc_fb_test/mxc_spdc_fb_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_fb_test/mxc_spdc_fb_test.c b/test/mxc_fb_test/mxc_spdc_fb_test.c
index 8c15337..ae2cd21 100644
--- a/test/mxc_fb_test/mxc_spdc_fb_test.c
+++ b/test/mxc_fb_test/mxc_spdc_fb_test.c
@@ -79,7 +79,7 @@ extern "C"{
 #define ALLOW_COLLISIONS	0
 #define NO_COLLISIONS		1
 
-#define NUM_TESTS		14
+#define NUM_TESTS		12
 
 __u32 pwrdown_delay = 0;
 __u32 scheme = UPDATE_SCHEME_QUEUE_AND_MERGE;
-- 
1.8.0

